perllib/FixMyStreet/App.pm

```diff
@@ -348,9 +347,19 @@ sub send_email {
 }
 
 sub send_email_cron {
-    my ( $c, $params, $env_from, $env_to, $nomail, $cobrand, $lang_code ) = @_;
-
-    return 1 if $c->is_abuser( $env_to );
+    my ( $c, $params, $env_from, $nomail, $cobrand, $lang_code ) = @_;
+
+    my $first_to;
+    if (ref($params->{To}) eq 'ARRAY') {
+        if (ref($params->{To}[0]) eq 'ARRAY') {
+            $first_to = $params->{To}[0][0];
+        } else {
+            $first_to = $params->{To}[0];
+        }
+    } else {
+        $first_to = $params->{To};
+    }
+    return 1 if $c->is_abuser($first_to);
```

Comment:
On a side note, do we do all this so we don't send questionnaires or the like to people in the abuse table?